pems/routes.py

```diff
@@ -0,0 +1,19 @@
+class Routes:
+    """Django routes in the form of `app:name` for the PeMS application."""
+
+    @property
+    def DISTRICTS_INDEX(self):
+        """Landing page for a Caltrans District."""
+        return "districts:index"
+
+    def to_dict(self) -> dict[str, str]:
+        """Get a mapping of property name --> value for each `@property` in the Routes collection."""
+        return {prop: str(getattr(self, prop)) for prop in dir(Routes) if isinstance(getattr(Routes, prop), property)}
+
+    @staticmethod
+    def name(route: str) -> str:
+        """Returns just the name portion of the app:name structure of the route."""
+        return route.split(":")[-1]
```

Comment:
I like that you're creating this `Routes` class like we have done in `Benefits` and `DDRC`. 

This style is a little verbose though (more the `Benefits` style that we started with).

Can you take a look at the [`cdt-identity` routes](https://github.com/Office-of-Digital-Services/django-cdt-identity/blob/main/cdt_identity/routes.py) and model this more on that? 

I'm also thinking _each app_ should define their own routes, and then we can aggregate those up into a common place (like in your context processor). We don't want to repeat the app names (e.g. `districts:index` here) all over the place if we don't have to.